### PR TITLE
Fix todo list width and overflow problem. Add color styling to app.

### PR DESCRIPTION
**Pull request by lim-is-a**

### CODE REVIEW

File: src/pages/TodoList.tsx [hunk 1]
```diff
@@ -7,11 +7,12 @@ import styled from 'styled-components'
...
-
-  ul {
-    padding-left: 0;
-  }
+  margin: 10px 0px 10px 10px;
+  padding-left: 6px;
+  background: yellow;
+  border-radius:15px;
+  width: 100%;
+  // overflow: scroll;
```
Comment (rgora23) [inline src/pages/TodoList.tsx:15]: Can we get rid of this comment?

File: src/pages/TodoList.tsx [hunk 2]
```diff
@@ -24,19 +25,25 @@ const TodoListItem = styled.div`
...
+  display: flex;
+  overflow: scroll;
+`
 
 const TodoListBody = styled.div`
-  display: flex;
-  justify-content: center;
-  max-width: 50%;
-  overflow: text-wrap;
+  // display: flex;
```
Comment (rgora23) [inline src/pages/TodoList.tsx:34]: looks like we can remove this

File: src/pages/TodoList.tsx [hunk 3]
```diff
@@ -24,19 +25,25 @@ const TodoListItem = styled.div`
     }
   }
 `
+const ListItem = styled.div`
+  display: flex;
+  overflow: scroll;
+`
```
Comment (rgora23) [inline src/pages/TodoList.tsx:31]: I tested this out locally and it looks great but the only thing I'd say is missing is some padding at the end of a really long string. 
I think you can just add `padding-right: 20px` here

**Review by rgora23 — CHANGES_REQUESTED** [review]

File: src/pages/TodoList.tsx (hunk 3 above)
Comment (lim-is-a) [reply src/pages/TodoList.tsx:30]: I thought I had but i'll make sure.

**Review by rgora23 — APPROVED** [review]

**Pull request merged**